tools/topology/topology2/get_abi.sh

```diff
@@ -11,6 +11,14 @@ cat <<EOF_HEADER
 Object.Base.manifest."sof_manifest" {
 	Object.Base.data."SOF ABI" {
 EOF_HEADER
+
+# ABI major/minor/patch numbers are 2 bytes each
 printf '\t\tbytes\t"0x%02x,' "$ABI_MAJOR"
+ABI_MAJOR=$(($ABI_MAJOR >> 8))
+printf '0x%02x,' "$ABI_MAJOR"
```

Comment:
Have you tested this? I tried it and it does not seem to truncate:

```shell
printf '0x%02x\n' 0xabcd
=> 0xabcd
```

This seems to work:

```shell
print_two_bytes () 
{ 
    local u16="$1";
    printf '0x%02x, 0x%02x,\n' $(((u16 & 0xff00) >> 8)) $((u16 & 0xff))
}

$ print_two_bytes 0xabcd
 => 0xab, 0xcd,

```

Reply:
test only with the current ABI which isnt more than 1 byte. let me fix this